### PR TITLE
RD-3737: Display task list on starting installation page appropriately

### PR DESCRIPTION
**Pull request by jawamartus**
## Description
N/A

## Screenshots / Videos
obraz

## Checklist

- [x] My code follows the UI Code Style.
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires a documentation update.
- [x] I added proper labels to this PR.

## Tests
N/A

## Documentation
N/A

### CODE REVIEW

File: app/components/GettingStartedModal/steps/SummaryStep/SummaryStep.tsx [hunk 1]
```diff
@@ -134,7 +134,7 @@ const SummaryStep = ({
                 blueprintsInstallationTasks.tasks) && (
                 <>
                     <Header as="h4">{i18n.t('gettingStartedModal.summary.taskListTitle')}</Header>
-                    <List ordered relaxed style={{ margin: 0, flex: 1, overflow: 'auto' }}>
+                    <List relaxed style={{ margin: 0, flex: 1, overflow: 'auto' }}>
```
Comment (qooban) [inline app/components/GettingStartedModal/steps/SummaryStep/SummaryStep.tsx:137]: Optional:

Code change looks good. It seems the numbers were never used here. Simple and doing the trick 🙂 
I'm wondering if we can remove `style` prop at all?

**Review by qooban — COMMENTED** [review]
Code change looks good 👍🏻 Maybe we can go even further (see comment).

Let's discuss details during our meeting today.

File: app/components/GettingStartedModal/steps/SummaryStep/SummaryStep.tsx (hunk 1 above)
Comment (jawamartus) [reply app/components/GettingStartedModal/steps/SummaryStep/SummaryStep.tsx:137]: I prefer to limit my changes to the JIRA ticket scope. I don't know the reason why style property was introduced here. For me it's too risky to remove it.

Comment (qooban) [reply app/components/GettingStartedModal/steps/SummaryStep/SummaryStep.tsx:137]: OK, let it be.

**Review by qooban — APPROVED** [review]
Seeing 
> I verified that all tests and checks have passed.
unticked, I'm wondering are you planning to verify it still somehow?

**Comment by jawamartus** [comment]
Tests passed for getting started modal.

https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/1874/pipeline/55

**Pull request merged**